keylime/src/agent_identity.rs

```diff
@@ -0,0 +1,353 @@
+use log::*;
+use openssl::x509::X509;
+use thiserror::Error;
+
+use crate::crypto::{x509_to_der, x509_to_pem, CryptoError};
+
+#[derive(Error, Debug)]
+pub enum AgentIdentityBuilderError {
+    /// Agent public AK not set
```

Comment:
Could you please remove these kind of comments? Error is already self-describing

Reply:
In case it is used for code documentation, I am OK with it